app.js

```diff
@@ -297,6 +415,18 @@ app.post('/hubs/:hubSlug/start_activity', function(req, res){
   }
 })
 
+app.post('/hubs/:hubSlug/devices/:deviceSlug/activate_command', function(req, res){
```

Comment:
This will also be more closer to the MQTT topic for commanding things:

`harmony-api/hubs/:hub_slug/devices/:device_slug/command` `power-on`

So that's nice too.